setup.py

```diff
@@ -22,7 +22,7 @@
 AUTHOR_EMAIL = "jarq6c@gmail.com"
 
 # Namespace package version
-VERSION = "1.0.0"
+VERSION = "1.2.0"
```

Comment:
Skipping `1.1.0`? We should probably already be there, just curious.

Reply:
Yeah, I wasn't sure. It should probably have been updated to 1.1.0 with `gcp_client`

Reply:
We can bump to `1.2.0` that's fine. That's my fault for not having bumped earlier.